### ISSUE TITLE
Adding scrollbar to window

### ISSUE THREAD

**Issue by Opadera** [wolfyxk/Foxhole-NoSpotterArty#2]
On my laptop screen the Tkinter window gets to big to display on my laptop screen.
There's an option for a [scrollbar ](https://www.tutorialspoint.com/python/tk_scrollbar.htm) in tkinter. Can you maybe add it or tell me what variable the "master window" is so I can try adding it?

thanks for your work! really like it

**Comment by jc-gc** [comment]
Yeah I can do, I've just picked this project back up after a long time and i'm planning on rewriting it, possibly with a different ui framework so I'm hoping to get something more polished this time.

Feel free to get your own version working in the meantime, I could be wrong since I've not worked with Tkinter in a long time, but I think it's self.mainwindow you are talking about to add the scrollbar.

I've also just started work on a C# version which im probably going to be spending a lot more time on and I'm planning on getting that finished first.
(https://github.com/wolfyxk/foxhole-intelarty)
